business/services/sale_offer.py

```diff
@@ -156,22 +156,27 @@ def change_sale_offer_status(sale_offer_status_excel, sale_offer_reference):
 
     sale_offer_status = None
 
-    if sale_offer_status_excel == 'DISABLED':
-        # Disabled the sale_offer
-        sale_offer_status = SaleOfferStatus('DISABLED')
-    elif sale_offer_status_excel == 'ARCHIVED':
-        # Archived the sale_offer
-        sale_offer_status = SaleOfferStatus('ARCHIVED')
-    elif sale_offer_status_excel == 'HOLIDAY':
-        # Holiday the sale_offer
-        sale_offer_status = SaleOfferStatus('HOLIDAY')
+    if sale_offer_status_excel is not None:
+        uppercase_status = sale_offer_status_excel.upper()
+        if uppercase_status == 'DISABLED':
+            # Disabled the sale_offer
+            sale_offer_status = SaleOfferStatus('DISABLED')
+        elif uppercase_status == 'ARCHIVED':
+            # Archived the sale_offer
+            sale_offer_status = SaleOfferStatus('ARCHIVED')
+        elif uppercase_status == 'HOLIDAY':
+            # Holiday the sale_offer
+            sale_offer_status = SaleOfferStatus('HOLIDAY')
 
     if sale_offer_status is not None:
         logging.info(f'Sale offer {sale_offer_reference} status change it {sale_offer_status_excel}')
```

Comment:
`Sale offer {sale_offer_reference} status change it {sale_offer_status_excel}` -> `Change sale offer {sale_offer_reference} status to {sale_offer_status_excel}`